knx/dpt/types_251_test.go

```diff
@@ -0,0 +1,32 @@
+package dpt
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestDPT_251600(t *testing.T) {
+	var buf []byte
+	var dst DPT_251600
+	var sources []DPT_251600
+
+	sources = append(sources, DPT_251600{Red: 255, Green: 96, Blue: 0, White: 18, RedValid: true, GreenValid: true, BlueValid: true, WhiteValid: true})
+	sources = append(sources, DPT_251600{Red: 255, Green: 96, Blue: 0, White: 18, RedValid: false, GreenValid: false, BlueValid: false, WhiteValid: false})
+
+	sources = append(sources, DPT_251600{Red: 255, Green: 96, Blue: 0, White: 18, RedValid: false, GreenValid: true, BlueValid: true, WhiteValid: true})
+	sources = append(sources, DPT_251600{Red: 255, Green: 96, Blue: 0, White: 18, RedValid: true, GreenValid: false, BlueValid: true, WhiteValid: true})
+	sources = append(sources, DPT_251600{Red: 255, Green: 96, Blue: 0, White: 18, RedValid: true, GreenValid: true, BlueValid: false, WhiteValid: true})
+	sources = append(sources, DPT_251600{Red: 255, Green: 96, Blue: 0, White: 18, RedValid: true, GreenValid: true, BlueValid: true, WhiteValid: false})
```

Comment:
Use a slice literal for `sources` instead.

Reply:
They look like this:

```go
sources := []DPT_251600{
  DPT_251600{Red: 255, Green: 96, Blue: 0, White: 18, RedValid: true, GreenValid: true, BlueValid: true, WhiteValid: true},
  DPT_251600{Red: 255, Green: 96, Blue: 0, White: 18, RedValid: false, GreenValid: false, BlueValid: false, WhiteValid: false},
  ....
}
```

Reply:
Totally makes sense as well. Will fix it.